bin/start_backend.sh

```diff
@@ -108,6 +111,15 @@ if [[ "$JAVA_VERSION" -gt 8 ]]; then
     fi
 fi
 
+# check NUMA setting
+NUMA_CMD=""
+if [[ "${RUN_NUMA}" -ne "-1" ]]; then
+    set -e
+    NUMA_CMD="numactl --cpubind ${RUN_NUMA} --membind ${RUN_NUMA}"
+    ${NUMA_CMD} echo "Running on NUMA ${RUN_NUMA}"
```

Comment:
if this numa node is not existed, we should exit here.

Reply:
`set -e` would make the script exit if numa node not exists